impl/src/main/java/impl/QuestionResourceImpl.java

```diff
@@ -149,9 +149,12 @@ private static MarkdownTextObject markdownText(String string, String... args) {
     }
 
     @Override
-    public Observable<List<Question>> getQuestions(long userId) {
+    public Observable<List<Question>> getQuestions(long userId, Integer limit) {
+        if (limit == null) {
+            limit = 10;
```

Comment:
An upper limit would be nice, so that someone doesn't take down the site just by spamming the API for all of the questions :)
This would actually be a sensible thing to have in the framework I believe, I don't see a case for not wanting an upper limit.

Reply:
CollectionOptions it is :)